.github/workflows/main.yml

```diff
@@ -0,0 +1,50 @@
+name: react-native-square-reader-sdk
```

Comment:
Can you please configure this repo to disable the travis CI check?

Reply:
Removed !!